### ISSUE TITLE
New to this - none of the examples seem to be working?

### ISSUE THREAD

**Issue by LordBastor** [draft-js-plugins/draft-js-plugins#684]
Hey guys,

Sorry if this seems like a stupid question - but are any of the examples in the docs working for you?

I'm following them to try and integrate draft-js for the first time in our project and I wanted to add some of the nifty plugins - like the inline or side toolbar plugin and the undo/redo button as well as some other styling tools which look really cool - but alas none of them seem to be triggering for me (using the latest version of Chromium and I even tried under firefox)

https://www.draft-js-plugins.com/plugin/side-toolbar
https://www.draft-js-plugins.com/plugin/inline-toolbar
The side and inline toolbar do nothing on click - both in my code and in the example:

Here is my code just to make sure I'm not doing something utterly stupid, but I'm trying for the minimum viable setup and it's not working (the Editor Itself is running though):
```
import React from 'react';
import ReactDOM from 'react-dom';
import {Editor, EditorState} from 'draft-js';

// Plugins
import createSideToolbarPlugin from 'draft-js-side-toolbar-plugin';
import 'draft-js-side-toolbar-plugin/lib/plugin.css';
const sideToolbarPlugin = createSideToolbarPlugin();


class DraftJSPage extends React.Component
{
	constructor(props)
	{
		super(props);
		this.state = {editor_state: EditorState.createEmpty()};
	}
	
	
	on_change(editor_state)
	{
		this.setState({editor_state});
	};

	focus()
	{
		console.log('focus');
		this.editor.focus();
	};
	
	render()
	{
		const styles = 
		{
			editor: {
				boxSizing: 'border-box',
				border: '1px solid #ddd',
				cursor: 'text',
				padding: '16px',
				borderRadius: '2px',
				marginBottom: '2em',
				boxShadow: 'inset 0px 1px 8px -3px #ABABAB',
				background: '#fefefe',
			}
		}
		
		return (
			<div style={styles.editor}>
				<Editor
					editorState={this.state.editor_state}
					onChange={this.on_change.bind(this)}
					plugins={[sideToolbarPlugin]}
					ref={(element) => { this.editor = element; }}
				/>
			</div>
		);
	}
}

export default DraftJSPage;
```

**Comment by richmeij** [comment]
When the plugin defines extra components (as is the case with sideToolbar and inlineToolbar), you need to extract the plugin components from the plugin and render them below the Editor.

So, for example for the sideToolbarPlugin, add this to the top of your file:
```
const sideToolbarPlugin = createSideToolbarPlugin();
const { SideToolbar } = sideToolbarPlugin; // <-- extract component from plugin
```

And then in your render, also render the SideToolbar component: 
```
    <Editor
        ...
    />
    <SideToolbar />
```

It works the same for inlineToolbar.

This is not very clear from the docs, but if you look at https://www.draft-js-plugins.com/plugin/side-toolbar and scroll past the initial example, you will see the explanation there.

**Comment by LordBastor** [comment]
You are completely correct - my first mistake (which I tried fixing yesterday but it didn't work) - was not extracting the component.

But I also missed one other thing - I kept using the `draft-js` Editor, where I needed the `draft-js-plugins-editor`

Like so:
`import Editor, { createEditorStateWithText } from 'draft-js-plugins-editor';`

Thank you for the help, much appreciated! Sorry for making this a stack overflow-y type of issue!
